assets/helpers/contributions.js

```diff
@@ -122,3 +130,25 @@ export function errorMessage(
   }
 
 }
+
+function contribCamelCase(contrib: Contrib) {
```

Comment:
I would add a return type to this function.

Reply:
Good catch, done/